optimum/habana/sentence_transformers/__init__.py

```diff
@@ -19,5 +19,5 @@
 from .st_gaudi_trainer import SentenceTransformerGaudiTrainer
 from .st_gaudi_training_args import SentenceTransformerGaudiTrainingArguments
 from .st_gaudi_encoder import st_gaudi_encode
-from .st_gaudi_transformer_tokenize import st_gaudi_transformer_tokenize
+from .st_gaudi_transformer_tokenize import st_gaudi_transformer_tokenize, st_gaudi_transformer_save
```

Comment:
```suggestion
from .st_gaudi_transformer_tokenize import st_gaudi_transformer_tokenize
from .st_gaudi_transformer_save import st_gaudi_transformer_save
```

@nngokhale I do not think the save function should be in the `st_gaudi_transformer_tokenize`. Either create a new file `st_gaudi_transformer_save` or other suggestion is to create a new `st_gaudi_transformer` file and add both `tokenize` and `save` there

Reply:
Renamed st_gaudi_transformer_tokenize.py to st_gaudi_transformer.py